meson.build

```diff
@@ -57,6 +57,8 @@ jxl = dependency('libjxl', required: false)
 if not jxl.found() and (jxl_feature.auto() or jxl_feature.enabled())
   jxl = cc.find_library('libjxl', required: jxl_feature)
 endif
+# Non-Linux
+epoll = dependency('epoll-shim', required: false)
```

Comment:
Let's set the proper "required"?
Something like this:
```
epoll = dependency('epoll-shim', required: host_machine.system() == 'freebsd')
```

Reply:
epoll-shim is used at least on DragonFlyBSD for Wayland and MidnightBSD.
If condition would be large as Wayland and epoll-shim are not specific to FreeBSD, so "required: false" is the current best practice used in e.g. foot:
https://codeberg.org/dnkl/foot/src/commit/ffdac61e2a95/meson.build#L79